package.json

```diff
@@ -49,6 +49,7 @@
     "firebase-tools": "^8.1.1",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^25.4.0",
+    "ncp": "^2.0.0",
```

Comment:
We also have the `cpx` dev dependency, which is also a file copier. We should probably stick with one, I don't really have a preference

Reply:
Good call, Ill use cpx because it x sounds cooler than n